js_basics/task5-3.js

```diff
@@ -0,0 +1,9 @@
+let length = 25;
```

Comment:
why do you still have it here? You have solved this task in j`s_basics/task5-2.js ` file.

Reply:
I get it! For some reason, I thought that I should also display the area in task5-3.js 
I also wondered why I should print it a second time. 
Thanks, I fixed it.